textures/build.gradle

```diff
@@ -0,0 +1,75 @@
+plugins {
```

Comment:
Definitely a need for a consistent quoting/brackets commit here. Are we doing `implementation thing` or `implementation(thing)`, `""` or `''`?

Reply:
Ideally, if @Prouser123 agrees, it will be kotlin style syntax. When you cut and paste from different things, you get different syntax.